sendsmaily.php

```diff
@@ -250,3 +250,10 @@ function smaily_nojs_subscribe_callback() {
 }
 add_action( 'admin_post_nopriv_smly', 'smaily_nojs_subscribe_callback' );
 add_action( 'admin_post_smly', 'smaily_nojs_subscribe_callback' );
+
+function showUpgradeNotification($currentPluginMetadata, $newPluginMetadata){
+	echo '<p style="background-color: #d54e21; padding: 10px; color: #f9f9f9; margin-top: 10px">
+		<strong>This version is no longer being developed. Please update.</strong> ';
+}
+// add plugin upgrade notification
+add_action('in_plugin_update_message-sendsmaily-wordpress-plugin/sendsmaily.php', 'showUpgradeNotification', 10, 2);
```

Comment:
The issue with using only this hook is that the notice will not be show after the plugin has been updated. It should stay visible even after updating to 1.2.4.